spec/core/data_structures.md

```diff
@@ -199,8 +199,8 @@ Commit is a simple wrapper for a list of signatures, with one for each validator
 
 | Name       | Type                             | Description                                                          | Validation                                                                                               |
 |------------|----------------------------------|----------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------|
-| Height     | int64                            | Height at which this commit was created.                             | Must be > 0                                                                                              |
-| Round      | int32                            | Round that the commit corresponds to.                                | Must be > 0                                                                                              |
+| Height     | int64                            | Height at which this commit was created.                             | Must be >= 0                                                                                             |
```

Comment:
Maybe it's worth adding a note after the table explaining when this field is set to 0 (IIRC, it's on the first block, to signal the absence of signatures)

Reply:
Right.

Reply:
Mentioned this below, on the rules for the signatures.